requirements.txt

```diff
@@ -1,2 +1,3 @@
 mo-future
 pyparsing
+six
```

Comment:
I did not see where `six` is being used.  Did I miss something?

Reply:
it is used in formatting.py, line 16

Reply:
oh! good catch!  Please remove the `six` from `requirements.txt` and add it to `tests/requirements.txt`